SQL/Archive/17.1/2016-11-21-CreateUserSitesRelTable.sql

```diff
@@ -0,0 +1,21 @@
+WARNINGS;
```

Comment:
https://dev.mysql.com/doc/refman/5.7/en/show-warnings.html

"In the mysql client, you can enable and disable automatic warnings display using the warnings and nowarning commands, respectively, or their shortcuts, \W and \w (see Section 5.5.1.2, “mysql Commands”)."

Maybe we should consider adding it to all of our scripts?